### DISCUSSION TITLE
How do I remove the previous button from step 1?

### DISCUSSION THREAD

**Discussion by edu22snt** [hakimio/ngx-ui-tour#151] [Q&A]
How do I remove the previous button from step 1?

I need the previous button from step 1 not to appear, is this possible?

![step1](https://github.com/hakimio/ngx-ui-tour/assets/8997162/b21152c2-a769-437f-b62f-d8d1d6607cea)

**Comment by hakimio** [comment]
Duplicate: https://github.com/hakimio/ngx-ui-tour/discussions/150
